.env

```diff
@@ -1,2 +1,2 @@
-VITE_USE_DUMMY_DATA=false # false for real question data otherwise dummy question data 
+VITE_USE_DUMMY_DATA=true # false for real question data otherwise dummy question data 
```

Comment:
We want this eventually going to prod?

I'm not a big fan of this variable.

Maybe you can have it as a mode when `node_env` is `development` but having in a configurable env doesn't seem right.

Reply:
Good point. I'll make a separate ticket regarding changing the `node_env` to development and based on the `NODE_ENV`, we then select to use either the DUMMY data or the inject data.